avaframe/com1DFAPy/utilities.py

```diff
@@ -0,0 +1,93 @@
+"""
+    utility functions for com1DFA
+
+    This file is part of Avaframe.
+"""
+
+# Load modules
+import os
+import glob
+import logging
+import numpy as np
+
+# Local imports
+import avaframe.in3Utils.fileHandlerUtils as fU
+import avaframe.in2Trans.ascUtils as IOf
+from avaframe.in3Utils import cfgUtils
+from avaframe.in3Utils import logUtils
+
+
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def getInputData(avaDir, cfg):
+    """ Fetch input datasets required for simulation
+
+    Input:
+            Directory
+
+    Output:
```

Comment:
Why not use a general function? Or provide this function generally? It seems to be a general enough task...

Reply:
moved to in1Data